registry/server/common/services/entries/ApplicationEntry.ts

```diff
@@ -0,0 +1,98 @@
+import db from '../../../db';
+import SharedLib from '../../../sharedLibs/interfaces';
+import {ValidationFqrnError} from './error/ValidationFqrnError';
+import {AssetsDiscoveryProcessor} from '../assets/AssetsDiscoveryProcessor';
+import {AssetsValidator} from '../assets/AssetsValidator';
+import App, {partialAppSchema} from '../../../apps/interfaces';
+import {Entry} from './Entry';
+import {stringifyJSON} from '../json';
+import {NotFoundApplicationError} from './error/NotFoundApplicationError';
+
+export class ApplicationEntry implements Entry{
+
+    private entityName = 'apps' as const;
+
+    constructor(private identifier?: string) {
+    }
+
+    public async patch(params: unknown, { user }: { user: any }) {
+
+        if(!this.identifier) {
+            throw new ValidationFqrnError('Patch does not invoked because instance was initialized w/o identifier');
+        }
+
+        await this.verifyExistence(this.identifier);
+
+        const appDTO = await partialAppSchema.validateAsync(params, {
+            noDefaults: true,
+            abortEarly: true,
+            presence: 'optional'
+        });
+
+        if(!Object.keys(appDTO).length) {
+            throw new ValidationFqrnError('Patch does not contain any items to update');
+        }
+
+        const  appManifest = await this.getManifest(appDTO.assetsDiscoveryUrl);
+
+        const appEntity = {
+            ...appDTO,
+            ...appManifest,
+        };
+
+        await db.versioning(user, {type: this.entityName, id: this.identifier}, async (trx) => {
+            await db(this.entityName)
+                .where({ name: this.identifier })
+                .update(stringifyJSON([
```

Comment:
will this field cover l10n path tempalte?

Reply:
Yeap. This method travers keys and convert its value to strings so all fields that does not require stringification does not change own type